auto_nag/utils.py

```diff
@@ -558,3 +558,12 @@ def bug_handler(bug, data):
 
 def nice_round(val):
     return int(round(100 * val))
+
+
+def make_ni(mail):
```

Comment:
Why didn't you use `auto_needinfo` like other scripts?

Reply:
because I'm adding a needinfo to the reviewer.
The way it works currently  doesn't really allow to add this extra info in the data to PUT.
I thought that it'll be less risky to do that here rather than refactor some stuff which could impact several tools so I limit the risks to this file only.

Reply:
I'm not sure it's that risky, the change should be relatively easy, but up to you ;)